core/templates/dev/head/forms/formBuilder.js

```diff
@@ -637,10 +637,10 @@ oppia.factory('rteHelperService', [
 oppia.config(['$provide', function($provide) {
   $provide.decorator('taOptions', [
       '$delegate', '$modal', '$timeout', 'focusService', 'taRegisterTool',
-      'rteHelperService',
+      'rteHelperService', 'explorationContextService',
```

Comment:
FYI, if we do this, I think we'll need to include ExplorationContextService.js at the bottom of all pages. Can you remove it from existing pages and put it in base.html instead?